arbnode/espresso_caff_node.go

```diff
@@ -128,13 +128,16 @@ func NewEspressoCaffNode(
 		log.Crit("failed to create espressoTEEVerifierCaller", "err", err)
 		return nil
 	}
-
+	client, err := espressoClient.NewMultipleNodesClient(configFetcher().HotShotUrls)
+	if err != nil {
+		log.Crit("Failed to create hotshot client", "err", err)
```

Comment:
Same here, if we encounter an error we still pass what will be a nil client to the node but not prevent it's startup.

Reply:
![image](https://github.com/user-attachments/assets/fc01cd37-b7f7-40c9-86fd-9cf3bb19297e)